lib/msf/core/post/process.rb

```diff
@@ -11,6 +13,18 @@ def initialize(info = {})
     ))
   end
 
+  #
+  # Gets the `pid` of a specified program
+  #
+
```

Comment:
This extra space is unnecessary. Also the documentation is just a little misleading. It actually returns an array of *all* pids of the specified program. The way it's worded makes it sound like it only returns one.

Reply:
okay I'll change it!